pkg/vmprovider/providers/vsphere/virtualmachine/backup.go

```diff
@@ -19,81 +19,92 @@ import (
 	"github.com/vmware-tanzu/vm-operator/pkg/vmprovider/providers/vsphere/constants"
 )
 
+// BackupOptions contains the options for calling BackupVirtualMachine() func.
+type BackupOptions struct {
```

Comment:
I thought the usual pattern is to call this `Context`?  IOW, this would be `BackupVMContext` or `VMBackupContext`?

Reply:
Updated as suggested in [8e2f56b](https://github.com/vmware-tanzu/vm-operator/pull/257/commits/8e2f56bf61e1b3ef978a633479c4f7b34b7f09f0).